src/Pokemon.java

```diff
@@ -0,0 +1,121 @@
+package src;
+
+public abstract class Pokemon {
+    private double healthPoint;
+    private double experiencePoints;
+
+    private String name;
+    private int level;
+    private double height;
+    private double weight;
+    private double defence;
+    private double attack;
+    private double accuracy;
+    private String special;
+
+
+    public abstract void speaks();
+    public abstract void eats(String food);
+    public abstract void attack();
+
+    public void getStats(){
```

Comment:
Dit is een leuke feature. Uitprinten wat de pokemon precies is. Je had hiervoor ook goed de "toString()" methode kunnen overwriten. Nu kun je dit printen door "sout(pikachu.getStats()", maar als je de toString overschrijft, kun je het printen als "sout(pikachu)". Dat komt omdat toString de default print methode is van de Object klasse